index.html

```diff
@@ -19,10 +19,24 @@
       <nav class="navbar">
         <a class="nav-logo" href="index.html">Mahdi Noori</a>
         <ul class="nav-list">
-          <li><a href="#">Portfolio</a></li>
-          <li><a href="#contact">Contact</a></li>
-          <li><a href="#about">About</a></li>
+          <li class="nav-link"><a href="#">Portfolio</a></li>
```

Comment:
![image](https://github.com/mahdinoori2000/portfolio/assets/93039359/4892261c-3122-4722-9a31-ae4729fe9ea6)

- [ ] It would be nice if this link shown in the image above should take me to the container of the project:+1: to follow the project requirements :blush: